lib/index.d.ts

```diff
@@ -37,23 +37,6 @@ export type ServiceClientOptions = {
     timeout?: number;
 };
 
-export interface ServiceRequest {
-    headers: Headers;
-    logger: Logger;
-    auth: {
-        isAuthenticated: boolean;
-        credentials: {
-            apiToken: string;
-            principalToken: string;
-        };
-    };
-}
-
-export type ServiceContext = {
-    dataSources: {[serviceClient: string]: ClientInstance};
```

Comment:
This would require use of [graphql-component](https://github.com/ExpediaGroup/graphql-component) which is not a requirement to use this library